### ISSUE TITLE
Failed to compile 

### ISSUE THREAD

**Issue by Elektroingenieur** [Daniel-Diaz/HaTeX#125]
Here's my specs:
- ghc-8.6.3
- stack Version  1.9.3

```

> HaTeX-3.1.0: download
> HaTeX-3.1.0: configure
> HaTeX-3.1.0: build
> Progress 1/2
> 
> --  While building package HaTeX-3.1.0 using:
>       /Users/haskell/.stack/setup-exe-cache/x86_64-osx/Cabal-simple_mPHDZzAJ_2.4.0.1_ghc-8.6.3 --builddir=.stack-work/dist/x86_64-osx/Cabal-2.4.0.1 build --ghc-options " -ddump-hi -ddump-to-file -fdiagnostics-color=always"
>     Process exited with code: ExitFailure 1
>     Logs have been written to: /Users/haskell/Projects/Haskell Projects/Thesis/.stack-work/logs/HaTeX-3.1.0.log
> 
>     Configuring HaTeX-3.1.0...
>     Preprocessing library for HaTeX-3.1.0..
>     Building library for HaTeX-3.1.0..
>     [ 1 of 28] Compiling Text.LaTeX.Base.Parser ( Text/LaTeX/Base/Parser.hs, .stack-work/dist/x86_64-osx/Cabal-2.4.0.1/build/Text/LaTeX/Base/Parser.o )
>     [ 2 of 28] Compiling Text.LaTeX.Base.Syntax ( Text/LaTeX/Base/Syntax.hs, .stack-work/dist/x86_64-osx/Cabal-2.4.0.1/build/Text/LaTeX/Base/Syntax.o )
>     
>     /private/var/folders/q6/kxf58_ts67n1syqwf8yg56dc0000gn/T/stack8221/HaTeX-3.1.0/Text/LaTeX/Base/Syntax.hs:54:10: error:
>         • No instance for (Semigroup LaTeX)
>             arising from the superclasses of an instance declaration
>         • In the instance declaration for ‘Monoid LaTeX’
>        |
>     **54 | instance Monoid LaTeX where**
>        |          ^^^^^^^^^^^^
> 
```
```

**Comment by leftaroundabout** [comment]
This problem should be fixed since #114. Please install an up-to-date version, 3.1.0 is positively ancient...

**Comment by Daniel-Diaz** [comment]
@Elektroingenieur Were you able to compile it?
